src/Template/Element/sort_option.ctp

```diff
@@ -0,0 +1,17 @@
+<md-menu-item>
+    <md-button>
+        <md-icon>
+            <?php 
+                if (strcmp($this->Paginator->sortKey(),$param) == 0 && strcmp($this->Paginator->sortDir(),$direction) == 0) {
+                    echo 'check'; 
+                }
+                else {
+                    echo 'blank';
+                }
+            ?>
+        </md-icon>
+        <span style="padding-right: 12px">
```

Comment:
Maybe move the CSS styling to `webroot/css/layouts/elements.css`?

Reply:
I forgot this tomorrow I will change it.

Reply:
I added a sortOption class to css file.